### ISSUE TITLE
Query exception with content type 'Controller'

### ISSUE THREAD

**Issue by DirkThiele** [zikula-modules/Content#308]
With debugging activated I sometimes get the following error message.

`[Semantical Error] line 0, col 74 near 'tblCategories.category': Error: 'tblCategories' is not defined.`

Refreshing the same page fixes the problem, sometimes I have to do it twice.

The error message only appears if there ist content of type `Controller` to be displayed.

In my case it is a list view for an entity, filtered by the ID of a category from the CategoriesModule with parameter `catId`.

For the `ContentModule` I have no category from the `CategoriesModule` assigned at all.

Not sure whether it is caused by the ContentModule / CategoriesModule or by the module I have built with MOST (1.4.0).

My installation: Content 5.0.1 / Zikula 2.0.11 / Symfony 3.4.14 / PHP 7.1.19-nmm1

**Comment by Guite** [comment]
This is a follow-up to #307.

Please enable your debug mode and post a complete stack trace for the exception.

**Comment by Guite** [comment]
I see the root cause of the problem (related to the quick navigation form in your MOST-based module). But wouldn't it be an option to use the module's dedicated list content type instead of the controller content type?

**Comment by Guite** [comment]
Closing as invalid (no Content bug). Either use the list content type from your module like suggested above and/or send me the custom MOST module via mail so I can take a look.

**Comment by DirkThiele** [comment]
With the Controller content type I simply wanted to add the `own=1` Parameter which I can't do with the list content type. Now I figured out that I can add this to the `for loop function` with an if-clause inside of the itemlist template.
Another aspect was, that with the `itemlist_myEntity_display.html.twig` the behaviour for some reason is different. For example my stylesheets are being ignored. With a custom template of the view.html.twig this problem would have been solved.

But in the end it really doesn't matter because this error just comes up if debugging is activated. And this will be turned off when the site is ready to use. ;-)
